### DISCUSSION TITLE
UseContractRead Making a request to my node every few seconds

### DISCUSSION THREAD

**Discussion by dobbobalina2** [wevm/wagmi#539] [Q&A]
Here are the setitngs I have. It is updating the parameters way too often. Any help would be appreciated. 

```
cacheTime: 2_000000,
        watch: true,
        enabled: false,
```

**Comment by tmm** [comment]
Setting `watch` to `true` either uses the client's [WebSocket provider](https://wagmi.sh/docs/client#websocketprovider-optional=) (if configured) or the regular [provider](https://wagmi.sh/docs/client#provider-optional=) in [polling mode](https://docs.ethers.io/v5/api/providers/provider/#BaseProvider). Either set a WebSocket provider or reduce the polling time.
